### PR TITLE
Add blog post on service intance sharing

### PR DESCRIPTION
**Pull request by mheadd**
## Changes proposed in this pull request:
- Add new blog post on service instance sharing

:sunglasses: [PREVIEW URL](https://cg-88d42ca6-59d7-47e0-9500-4dd9251360b9.app.cloud.gov/preview/cloud-gov/cg-site/mheadd-patch-30/2022/02/09/sharing-service-instances/)

## Security Considerations
None

### CODE REVIEW

**Comment by mheadd** [comment]
cc: @mogul since you made some of the changes need to support this.

File: _posts/2022-02-09-sharing-service-instances.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+In the past, we have heard from developers that have wanted to extend this functionality further - to allow service instances to be shared across different spaces in a cloud.gov organization. This was something that the cloud.gov platform did not support. 
+
+Until today.
+
+### Sharing is caring
+
+We are happy to announce that [service instance sharing](https://docs.cloudfoundry.org/devguide/services/sharing-instances.html#sharing) has now been enabled on the cloud.gov platform. Here's how it works:
+
+* A developer uses the standard `cf create-service` command to instantiate a new service instance in a particular space.
+* When initially created, viewing the service details via `cf service {service_name}` will indicate that the service is not current shared with any other spaces (see below).
```
Comment (rbogle) [inline _posts/2022-02-09-sharing-service-instances.md]:
```suggestion
* When initially created, viewing the service details via `cf service {service_name}` will indicate that the service is not currently shared with any other spaces (see below).
```

**Review by rbogle — DISMISSED** [review]
:shipit: 👍

**Review by rbogle — APPROVED** [review]

**Comment by mogul** [comment]
I love it. Now I'm waiting on the follow-up articles about some of the use-cases I posted in Slack... ;)

image

**Pull request merged**